metrics/snapshot-collector/README.md

```diff
@@ -1,16 +1,20 @@
-snapshot-collector
+metrics-snapshot-collector
 ------
-**This directory for capture Grafana's URLs to genertate images**
+**This tool is used to capture screenshots of grafana**
+### Notice
+- **Ensure grafana server have shared object file `libfontconfig.so.1`**
```

Comment:
Make sure that the shared library `libfontconfig.so.1` is installed on the Grafana server in advance.